src/RA_Defs.h

```diff
@@ -63,6 +52,23 @@ extern GetParseErrorFunc GetJSONParseErrorStr;
 #pragma warning(pop)
 #endif	//RA_EXPORTS
 
+// Maybe an extra check just in-case
+
+#define _NORETURN            [[noreturn]]
+
+#if _HAS_CXX17
+#define _DEPRECATED          [[deprecated]]
+#define _DEPRECATEDR(reason) [[deprecated(reason)]]
+#define _FALLTHROUGH         [[fallthrough]]//; you need ';' at the end
+#define _UNUSED              [[maybe_unused]]
+#else
+#define _NODISCARD           _Check_return_
+#define _DEPRECATED          __pragma("This feature is deprecated") 
+#define _DEPRECATEDR(reason) __pragma(reason)
+#define _FALLTHROUGH         __fallthrough//; you need ';' at the end
+#define _UNUSED              
+#endif // _HAS_CXX17
+
 
 #define RA_KEYS_DLL						"RA_Keys.dll"
 #define RA_PREFERENCES_FILENAME_PREFIX	"RAPrefs_"
```

Comment:
Before anyone says it's not aligned, it is. Click "View" and it will look aligned as well as in Visual Studio.

Reply:
you can click and drag the whitespace between the #define and the value. there's TABs in there...

Reply:
well I'm pretty sure I showed you already but here goes.
GitHub:
![image](https://user-images.githubusercontent.com/18649925/40593310-bc54bf2a-61f5-11e8-834e-84fdf3eeb213.png)

Visual Studio:
![image](https://user-images.githubusercontent.com/18649925/40593394-44e7f802-61f6-11e8-9ab5-f4fb64fcc57b.png)

Reply:
I tried converting tabs to spaces but it still looks misaligned on sourcetree.